keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -156,6 +156,9 @@ parameters" KEP](../4381-dra-structured-parameters/README.md) added an
 extension. Now the roles are reversed: #4381 defines the base functionality
 and this KEP is an optional extension.
 
+In Kubernetes 1.32, this KEP has been **withdrawn** and all code related to it
```

Comment:
Can you make this more obvious?

Memory-QoS is a good example to follow.

https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos#latest-update-stalled.

We still have people asking about the feature but it would be good to have a section to state that we are no longer supporting this KEP.

Reply:
What should be more obvious? "All code related to it gets removed" means it'll be gone.

Reply:
The KEP is a record for design decisions. Most people will not read the code and they may come to the KEP to get information about this. I think this important detail should be clearly stated about why we removed this feature. 

I would prefer a section near the top of this document that states why we made the decision to remove classical DRA so it is easy to reference. We can't really reference deleted code.

Reply:
I see: you wanted more information about the background, not about what "withdrawn" means.

I've added the two two items which came up most often (autoscaling, PodSchedulingContext).

Reply:
Thank you. This looks good to me.